test-cases/artifacts/ami.yaml

```diff
@@ -15,3 +15,5 @@ scylla_linux_distro: 'centos'
 test_duration: 60
 user_prefix: 'artifacts-ami'
 system_auth_rf: 1
+
+scylla_mgmt_agent_repo: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.4.repo'
```

Comment:
The ami default should be ubuntu 20.04.

Also, please make sure that it doesn't conflicts with the jenkins pipeline parameter.